### ISSUE TITLE
False triggered

### ISSUE THREAD

**Issue by ncurran02** [Hockeyfan360/Capatchafy#13]
I have joined and left a few times due me lagging out, and the capatchafy plugin thought the server was under attack.

**Comment by Hockeyfan360** [comment]
Yeah, it's because it throttles logins with a syncronous task. I've got several projects due this week, and then finals the following week, so I might not get to a patch soon. However, once winter break begins, I should have enough time to look into this.
